### PR TITLE
Adds border attributes to Column, Section and Container

### PR DESCRIPTION
**Pull request by bandrzejczak**
This is basically implementation of PR #103 applied to version 2.2.0.

### CODE REVIEW

File: packages/mjml-section/src/index.js [hunk 1]
```diff
@@ -117,6 +117,21 @@ class Section extends Component {
       div: this.isFullWidth() ? {} : cloneDeep(background),
       table: this.isFullWidth() ? {} : cloneDeep(background),
       tableFullwidth: this.isFullWidth() ? cloneDeep(background) : {}
+    }, {
+    	table: this.isFullWidth() ? {} : {
+				border: mjAttribute('border'),
```
Comment (iRyusa) [inline packages/mjml-section/src/index.js]: Can you convert tabs to space please ?

**Comment by bandrzejczak** [comment]
@iRyusa Could you run the travis build again? It somehow didn't catch my commit...

**Comment by bandrzejczak** [comment]
@iRyusa Done

**Comment by iRyusa** [comment]
Hello  @bandrzejczak and thanks for your contribution !

Can you update your PR with the documentation and some screenshot from litmus ? Please check the https://github.com/mjmlio/mjml/blob/master/CONTRIBUTING.md !

**Comment by blaugueux** [comment]
+10 000 👍

**Comment by iRyusa** [comment]
Hello there,

Quick update, i've added border on #321 for section/column but i can't add it on `container` yet. Render is totally broken on older outlook

**Comment by iRyusa** [comment]
Hi @bandrzejczak 

Same as #103 `border` has been added into MJML3 on section & column. I'm sorry to close this PR for now but thanks for your contribution. Border is now OK on every clients on Litmus in MJML3 !

**Comment by bandrzejczak** [comment]
@iRyusa Wow! Awesome. Thanks for the heads up! 👍

**Pull request closed without merging**